test/workers/send_user_invitation_worker_test.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class SendUserInvitationWorkerTest < ActiveJob::TestCase
+  def test_provider_mailer_perform
+    Timecop.freeze(Time.zone.parse('2017-11-23 03:25:08 UTC +00:00')) do
+      ProviderInvitationMailer.any_instance.expects(:invitation).once
+
+      provider_account = FactoryBot.create(:provider_account)
+      invitation = FactoryBot.create(:invitation, account: provider_account)
+
+      assert_nil invitation.sent_at
+      SendUserInvitationWorker.new.perform(invitation.id)
+      assert invitation.reload.sent_at, Time.zone.now
```

Comment:
```suggestion
      assert_equal invitation.reload.sent_at, Time.zone.now
```

Reply:
assert equal for times/dates is a bit asking for problems... even inside the timecop. I would suggest:
```suggestion
      assert_in_delta Time.zone.now, invitation.reload.sent_at, 1.second
```